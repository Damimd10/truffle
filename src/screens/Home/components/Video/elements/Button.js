import styled from 'styled-components';

const Button = styled.button`
  background-color: #30393c;
  color: #fff;
  border-radius: 5px;
  height: 25px;
  padding: 0 20px;
  &:hover {
    cursor: pointer;
  }
  &:focus {
    outline: none;
  }
  &:disabled {
    cursor: default;
    background-color: #797e80;
  }
`;

export default Button;
