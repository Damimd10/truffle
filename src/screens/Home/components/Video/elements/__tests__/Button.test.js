import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Button from '../Button';

describe('Button Style Element', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Button />);
  });

  it('should create a style structure for the component', () =>
    expect(toJson(wrapper)).toMatchSnapshot());
});
