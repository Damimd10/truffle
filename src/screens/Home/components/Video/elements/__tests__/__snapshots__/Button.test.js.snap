// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Style Element should create a style structure for the component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  background-color: #30393c;
  color: #fff;
  border-radius: 5px;
  height: 25px;
  padding: 0 20px;
  &:hover {
    cursor: pointer;
  }
  &:focus {
    outline: none;
  }
  &:disabled {
    cursor: default;
    background-color: #797e80;
  }
",
        ],
      },
      "displayName": "styled.button",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "button",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
